### ISSUE TITLE
Errors

### ISSUE THREAD

**Issue by maxmstrmn** [naman-mehta/predict_steering_angle#1]
LoadTrainBatch and LoadValBatch are missing. How can I use and test your script?
Please also note that this code is unusable with TF 2.0 as tf.InteractiveSession is unavailable

**Comment by maxmstrmn** [comment]
Please also note that this code is unusable with TF 2.0 as tf.InteractiveSession is unavailable

**Comment by naman-mehta** [comment]
Yes you are right. It is still not complete.  I will try to finish it by the end of this week and I will let you once it is completed. Thanks a lot for visiting my repository.

**Comment by maxmstrmn** [comment]
Hi, thank you very much for responding. I really look forward to some working code. I appreciate your work! Please keep me updated :D
